boilerplate/openshift/custom-catalog-osd-operator/standard.mk

```
@@ -10,7 +10,7 @@ REGISTRY_TOKEN?=$(QUAY_TOKEN)
 
 
 # Accommodate docker or podman
-CONTAINER_ENGINE=$(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)
+CONTAINER_ENGINE=$(shell command -v podman 2>/dev/null || echo docker --config=$(CONTAINER_ENGINE_CONFIG_DIR))
```

nit: This works fine, but would be "safer" with some quotes:
```suggestion
CONTAINER_ENGINE=$(shell command -v podman 2>/dev/null || echo "docker --config=$(CONTAINER_ENGINE_CONFIG_DIR)")
```